cmd/process-agent/collector_test.go

```diff
@@ -3,26 +3,31 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
+/*
+TODO:
+- Plug in NewSubmitter in tests and checks
+- Refactor tests into collector_tests and submitter_tests
+- Mock the submitter (gen_mocks)
+*/
+
```

Comment:
👍

Reply:
Thanks for catching this. I should have seen it when I looked over my PR 🙈